as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ShowInBrowserTest.java

```diff
@@ -0,0 +1,38 @@
+package org.jboss.tools.as.ui.bot.itests.parametized.server;
+
+import static org.junit.Assert.assertTrue;
+import org.eclipse.reddeer.eclipse.wst.server.ui.cnf.Server;
+import org.eclipse.reddeer.eclipse.wst.server.ui.cnf.ServersView2;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.server.ServerRequirementState;
+import org.eclipse.reddeer.swt.api.Browser;
+import org.eclipse.reddeer.swt.api.Menu;
+import org.eclipse.reddeer.swt.impl.browser.InternalBrowser;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenu;
+import org.jboss.ide.eclipse.as.reddeer.server.requirement.ServerRequirement.JBossServer;
+import org.jboss.tools.as.ui.bot.itests.AbstractTest;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(RedDeerSuite.class)
+@JBossServer(state=ServerRequirementState.RUNNING)
+public class ShowInBrowserTest extends AbstractTest {
```

Comment:
Why are you extending AbstractTest class? Do you use some of it?

Reply:
This test should have all options, which are in Show In context menu: 
![image](https://user-images.githubusercontent.com/17871936/130026780-50ac4cb2-93df-4566-9676-e2c56ddc43b4.png)

Right now only show in Web Browser is implemented.
I would also suggest to change class name to something like: ShowInContextMenuTest
(since it will have more test methods - not only web Browser)